Naveen-gali/Yelp-App
# Localization Setup

### What does this PR do?

- Add Localization Support for Yelp App.


### Related Issues

- closes #32 

### How to test

- Clone the Repository.
 - Switch to `feat/localizationSupport` branch.
- Do yarn to install the dependencies.
- Do yarn run build-stage-ios for ios or yarn run build-stage-android for android.
- You will notice a Welcome text on Screen and if you try changing the Language of the device. You can notice the change in the text language.

### Screenshots

#### Android Screen Shots
|English            |  Arabic |
|:-------------------------:|:-------------------------:
| ![Screenshot_1681105151](https://user-images.githubusercontent.com/122969303/230835007-f7bb7beb-f1b0-4f4f-8d59-3f4eda993713.png) |  ![Screenshot_1681117290](https://user-images.githubusercontent.com/122969303/230870103-d3620189-0732-4183-bd74-aa49a3f553ba.png) |

#### Ios Screen Shots
|English            |  Arabic |
|:-------------------------:|:-------------------------:
| ![Simulator Screen Shot - iPhone 14 - 2023-04-10 at 11 01 49](https://user-images.githubusercontent.com/122969303/230835145-8a3baa68-bf29-4746-a4f5-893fa32f32ed.png) |  ![Simulator Screen Shot - iPhone 14 - 2023-04-10 at 14 27 05](https://user-images.githubusercontent.com/122969303/230869357-adcc0ef6-57a1-4761-8b92-cef9fb1a1e4f.png) |

### Checklist


### Additional Notes

- [Change Language in Android Emulator](https://stackoverflow.com/questions/5755460/how-to-change-the-default-language-of-android-emulator)

### Packages Added (if any)

| Package | Reason |
| ------- | ------ |
|react-native-localization         | Package that adds support for Localization of content.        |
